plugins/acss-kit/assets/foundation/sass/_properties.scss

```
@@ -0,0 +1,107 @@
+  --fs-xl: var(--fs-9);
+  --fs-xxl: var(--fs-11);
+  --fs-xxxl: var(--fs-12);
+
+  --display: var(--fs-12);
+
+  --display-md: var(--fs-11);
+  --display-sm: var(--fs-10);
+
+  // h1
+  --h1: var(--fs-11);
+  // h2
```

Stylelint (17.10.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)